test/pools/PoolCollectionWithdrawal.ts

```diff
@@ -134,11 +134,11 @@ describe('PoolCollectionWithdrawal', () => {
         });
 
         test('PoolCollectionWithdrawalCoverage8', {
-            p: { maxAbsoluteError: new Decimal(0), maxRelativeError: new Decimal('0') },
-            q: { maxAbsoluteError: new Decimal(0), maxRelativeError: new Decimal('0') },
-            r: { maxAbsoluteError: new Decimal(0), maxRelativeError: new Decimal('0') },
+            p: { maxRelativeError: new Decimal('0') },
+            q: { maxRelativeError: new Decimal('0') },
+            r: { maxRelativeError: new Decimal('0') },
             s: { maxAbsoluteError: new Decimal(1), maxRelativeError: new Decimal('0'), relation: Relation.LesserOrEqual }, // prettier-ignore
-            t: { maxAbsoluteError: new Decimal(0), maxRelativeError: new Decimal('0') },
+            t: { maxRelativeError: new Decimal('0') },
```

Comment:
Well, in this case, I actually left them here on purpose, in order to keep a full depiction of the maximum error in each case. If you insist on removing the `maxAbsoluteError` default ones, then note that we can remove the `maxRelativeError` default ones as well.

Reply:
Oh, yeah. Let's remove them as well